docs/API.md

```diff
@@ -10,6 +10,7 @@ automatically logged in and validated against mojang's auth.
  * host : default to undefined which means listen to all available ipv4 and ipv6 adresses
  (see https://nodejs.org/api/net.html#net_server_listen_port_host_backlog_callback for details)
  * kickTimeout : default to `10*1000` (10s), kick client that doesn't answer to keepalive after that time
+ * timeoutReason : default to `Timed out`, the end reason that is sent to the client when the client doesn't answer to keepalive packets
```

Comment:
Can you say what is one use case of changing that error message?

Reply:
for example: translate to other launguage, etc

Reply:
if server is in bungeecord network to do not trigger the 'Lost connection to server' error from bungeecord

Reply:
because bungeecord checks if the message equals 'Time out'

Reply:
usage 3: if server is in bungeecord network that uses 'FlameCord' fork as serevr software to do not get player firewalled from the server

Reply:
i can provide more examples, if need

Reply:
> Can you say what is one use case of changing that error message?

Should I put 1 (or more) use cases in the documentation or not?

Reply:
> > Can you say what is one use case of changing that error message?
> 
> Should I put 1 (or more) use cases in the documentation or not?

no

Reply:
Why is it useful not to trigger the lost connection from server from bungeecord ?

Reply:
> Why is it useful not to trigger the lost connection from server from bungeecord ?

plugins can throw exceptions. and server console will be in lots of warnings

Reply:
> > Why is it useful not to trigger the lost connection from server from bungeecord ?
> 
> plugins can throw exceptions. and server console will be in lots of warnings

and if bungeecord proxy runs flamecord then the player will get firewalled (temp banned) from the server, if the plugin throw exception

Reply:
> Why is it useful not to trigger the lost connection from server from bungeecord ?

and the lost connection thing can trigger for lots of reasons (example: server shut down, bad packet from server, bad packet from client, etc), if u change the timeout reason then you can know why this error happened

Reply:
> usage 3: if server is in bungeecord network that uses 'FlameCord' fork as server software to do not get player firewalled from the server

only if readtimeoutexception on old flamecord was thrown*

Reply:
> > usage 3: if server is in bungeecord network that uses 'FlameCord' fork as server software to do not get player firewalled from the server
> 
> only if readtimeoutexception on old flamecord was thrown*

https://github.com/2lstudios-mc/FlameCord/commit/544492fb6382056cb9120cfd28a9e6054f488f43

Reply:
> i can provide more examples, if need

usage 4: customize the message for your server